code/game/objects/items/weaponry.dm

```diff
@@ -975,7 +975,7 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	force = initial(force)
 	armour_penetration = initial(armour_penetration)
 	if(isstructure(A) || ismachinery(A) || isturf(A))
-		force *= 4
+		force *= 2
```

Comment:
Split the multiplier up:

It should do good bonus damage to airlocks, barricades and such but not to walls. 

We may go back to RCDs being a problem but billy clubs are not the solution - it's finding a way to control RCDs in combat situations or players realizing that it really isn't that hard to make normal walls without one. Making it so attacking the field shorts out the RCD for a while